app.js

```diff
@@ -0,0 +1,18 @@
+const form = document.querySelector('form');
+const input = document.querySelector('input');
+
+
+addTask = (e) => {
+    e.preventDefault();
+    const titleTask =  input.value;
+    const ul = document.createElement('ul');
+    const li = document.createElement('li');
+    ul.appendChild(li);
+    document.body.appendChild(ul);
+    li.textContent =  titleTask;
```

Comment:
What if `titleTask ` is empty, or it's just one space, ore any other white character? I think in that case, it shouldn't be able to add item to list